src/components/Tooltip/Tooltip.tsx

```diff
@@ -17,6 +17,8 @@ export interface TooltipProps {
   light?: boolean;
   /** Toggle whether the tooltip is visible */
   active?: boolean;
+  /** Dismiss Tooltip once pointer is no longer over children */
+  preventInteraction?: TooltipOverlayProps['preventInteraction'];
```

Comment:
I think `preventInteraction` or `preventPointerEvents` makes sense for the prop name on `PositionedOverlay`, but for `Tooltip` and `TooltipOverlay` it doesn't describe clearly what the use case is or what is going on under the hood when set. My understanding from reading through Curtis' review comments is that the origin of the need for this prop lies in needing to allow interaction with the content of the tooltip. If that's correct, what do you think of something like `hasInteractiveContent`?

```suggestion
  hasInteractiveContent?: TooltipOverlayProps['preventInteraction'];
```

Reply:
dismissOnMouseOut